Checkout/Checkout/Source/Validation/Validators/CVVValidator.swift

```diff
@@ -23,14 +27,18 @@ final class CVVValidator: CVVValidating {
       return .failure(.containsNonDigits)
     }
 
-    guard let cvvLength = cardScheme.cvvLength else {
-      let isValidCVV = cvv.count == 3 || cvv.count == 4
-      return isValidCVV ? .success : .failure(.invalidLength)
-    }
-
-    return cvv.count == cvvLength ? .success : .failure(.invalidLength)
+      return cardScheme.cvvLengths.contains(cvv.count) ?
+          .success :
+          .failure(.invalidLength)
   }
-
+    
+    func isValid(cvv: String, for scheme: Card.Scheme) -> Bool {
+        return validate(cvv: cvv, cardScheme: scheme) == .success
```

Comment:
Do you need a return statement ? I thought swift will return back automatically if it has one line in a function, Same as your line 40.

Reply:
Nice one, thank you